hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -845,6 +847,11 @@ bool MDMParser::init(DevStatus* status)
     sendFormated("AT+CIMI\r\n");
     if (RESP_OK != waitFinalResp(_cbString, _dev.imsi))
         goto failure;
+    // Reformat the operator string to be numeric
+    // (allows the capture of `mcc` and `mnc`)
+    sendFormated("AT+COPS=3,2\r\n");
```

Comment:
Does this command affect the response of AT+COPS?  Does it still return <mode> 0, 2, or 4 if those are set and not 3?

Reply:
Here is a sample AT response:
```none
// +COPS: <mode>[,<format>,<oper>[,<AcT>]]
+COPS: 0,2,"310410",8
```

If you are asking what the behavior is like when you force set your <oper> using `AT+COPS=1,2`, then I don't know the answer. I would appreciate it greatly if you know how and are comfortable testing.

I don't have multiple pieces of hardware, and I'm afraid to twiddle with those for fear of messing up a device.

Reply:
This looks to be ok then with a AT+COPS=0 automatic mode selected.  I'm assuming if you tried manual AT+COPS=4,2,"310410" it would also return similarly +COPS: 4,2,"310410",8 but with the mode noted as 4 (manual).  Was just wanting to check that it didn't return mode = 3.

Reply:
We are now sending `AT+COPS=x,2`, so the format is constantly updated to be numeric.